packages/n/nautilus-open-any-terminal/package.yml

```diff
@@ -0,0 +1,20 @@
+name       : nautilus-open-any-terminal
+version    : 0.6.1
+release    : 1
+source     :
+    - git|https://github.com/Stunkymonkey/nautilus-open-any-terminal.git : 0.6.1
+homepage   : https://github.com/Stunkymonkey/nautilus-open-any-terminal
+license    : GPL-3.0-or-later
+component  : desktop.gnome
+summary    : nautilus-open-any-terminal is an extension for nautilus, which adds an context-entry for opening other terminal emulators than gnome-terminal.
+description: |
+    nautilus-open-any-terminal is an extension for nautilus, which adds an context-entry for opening other terminal emulators than gnome-terminal.
```

Comment:
Can we change it to read "An extension for nautilus, which adds an context-entry for opening other terminal emulators than gnome-terminal." ?